src/components/Views.tsx

```diff
@@ -19,6 +19,8 @@ type TViewItemProps = {
 
 type TViewsProps = {
     portalNodeId?: string;
+    onChartType: (chartType?: string | undefined) => void;
```

Comment:
Is this extra undefined needed here?
`onChartType: (chartType?: string) => void;`